packages/python/plotly/plotly/express/_doc.py

```diff
@@ -444,7 +446,7 @@
 
 
 def make_docstring(fn):
-    tw = TextWrapper(width=79, initial_indent="    ", subsequent_indent="    ")
+    tw = TextWrapper(width=77, initial_indent="    ", subsequent_indent="    ")
```

Comment:
not sure I understand this change...

Reply:
It's to prevent spillover in the help message that pops up when clicking shift + tab in Jupyter notebooks. I thought the width for that was 79 when I reformatted the docstrings previously, but noticed today that it actually is 77.

Before:
![image](https://user-images.githubusercontent.com/4560057/68128253-ab9c7900-ff17-11e9-9c06-806ae7125ba0.png)

After:
![image](https://user-images.githubusercontent.com/4560057/68128383-e7cfd980-ff17-11e9-9831-2877d3ce5097.png)

Reply:
OK thank you for the explanation! It's weird however because I don't have the problem 
![image](https://user-images.githubusercontent.com/263366/68129829-3e252400-fee8-11e9-9782-7c1b9daf8e59.png)

Reply:
Which version of jupyter are you using? Mine is 
```
$ jupyter --version
jupyter core     : 4.5.0
jupyter-notebook : 6.0.1
qtconsole        : 4.5.5
ipython          : 7.8.0
ipykernel        : 5.1.2
jupyter client   : 5.3.1
jupyter lab      : 1.1.4
nbconvert        : 5.6.0
ipywidgets       : 7.5.1
nbformat         : 4.4.0
traitlets        : 4.3.2

```

Reply:
But I'm using a dark theme for jupyter which might tune the configuration. I also tested on binder and there is even more spillover I think, it looks like it does not use wrapping at all? 
![image](https://user-images.githubusercontent.com/263366/68130312-2306e400-fee9-11e9-9040-51e7d9552cd0.png)

Reply:
This issue seems more complicated than I orginally thought, would you mind reverting here to 79 and opening an issue so that we can take the time to understand what's going on, and merge the rest of this PR?

Reply:
I think it is the custom dark theme that changes the width of the help box in the first screen shot (it looks like you have even more room on the right). I tried the default Jupyter Light and Dark theme and they both perform the wrapping as I indicated above:

![image](https://user-images.githubusercontent.com/4560057/68135634-48651380-ff24-11e9-8c16-7450f396380b.png)

On your binder screenshot, it appears as if you are running without PR #1835. I'm on jupyter 4.4.0 and jupyterlab 1.0.5.

Reply:
Do you prefer if I make a new commit that to revert the change, or can I drop the old one in a rebase and then force push?

Reply:
Thanks a lot for helping to document this. OK, so no need to revert the change, I'll merge as it is.